src/mdm-profile.resource.ts

```diff
@@ -100,4 +100,23 @@ export class MdmProfileResource extends MdmResource {
 
         return this.simpleDelete(url, queryStringParams, restHandlerOptions);
     }
+
+    validateProfile(
+        profileNamespace: string,
+        profileName: string,
+        catalogueItemDomainType: string | ModelDomainType,
+        id: string,
+        data: string,
+    )
+    {
+        let url = `${this.apiEndpoint}/profiles/${profileNamespace}/${profileName}/${catalogueItemDomainType}/${id}/validate`;
```

Comment:
Tiny thing, these `let` statements can be `const`. `eslint` would pick these up, but don't think it's really setup for this repo.